client/src/components/BoulderCard.stories.jsx

```diff
@@ -0,0 +1,30 @@
+import BoulderCard from './BoulderCard';
+import '../App.css';
+
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'components/Cards/BoulderCard',
+  component: BoulderCard,
+  // More on argTypes: https://storybook.js.org/docs/react/api/argtypes
+//   argTypes: {
+//     backgroundColor: { control: 'color' },
+ 
+};
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
```

Comment:
You should remove the comments as they don't seem to be necessary for your story.

Reply:
Done.